demos/graph-classification/supervised-graph-classification.ipynb

```diff
@@ -471,10 +521,12 @@
    "source": [
     "test_accs = []\n",
     "\n",
-    "for train_index, test_index in model_selection.RepeatedStratifiedKFold(\n",
+    "stratified_folds = model_selection.RepeatedStratifiedKFold(\n",
     "    n_splits=folds, n_repeats=n_repeats\n",
-    ").split(graph_labels, graph_labels):\n",
+    ").split(graph_labels, graph_labels)\n",
     "\n",
+    "for i, (train_index, test_index) in enumerate(stratified_folds):\n",
+    "    print(f\"Training and evaluating on fold {i+1} out of {folds * n_repeats}...\")\n",
```

Comment:
If we were concerned about the verbosity here, we _could_ go for:

```python
if (i + 1) % 5 == 0:
    print(f"Training and evaluating on fold {i+1} out of {folds * n_repeats}...")
```

and so only get every 5th update, but that's probably not necessary and a little confusing (we'd have to move it to the end of the loop and rephrase the message or something)